src/pudl/transform/eiaaeo.py

```diff
@@ -358,20 +358,115 @@ class AeoCheckSpec:
 
     name: str
     asset: str
-    num_rows_by_report_year: dict[str, int]
-    num_in_categories: dict[str, int]
+    num_rows_by_report_year: dict[int, int]
+    category_counts: dict[str, int]
 
 
+BASE_AEO_CATEGORIES = {
+    "model_case_eiaaeo": 17,
+    "projection_year": 30,
+    "electricity_market_module_region_eiaaeo": 26,
+}
 check_specs = [
     AeoCheckSpec(
         name="gen_in_electric_sector_by_tech",
         asset="core_eiaaeo__yearly_projected_generation_in_electric_sector_by_technology",
         num_rows_by_report_year={2023: 166972},
-        num_in_categories={"model_case_eiaaeo": 17, "projection_year": 30},
-    )
+        category_counts=BASE_AEO_CATEGORIES
+        | {
+            "technology_description_eiaaeo": 13,
+        },
+    ),
+    AeoCheckSpec(
+        name="gen_in_electric_sector_by_tech",
+        asset="core_eiaaeo__yearly_projected_generation_in_end_use_sectors_by_fuel_type",
+        num_rows_by_report_year={2023: 77064},
+        category_counts=BASE_AEO_CATEGORIES
+        | {
+            "fuel_type_eiaaeo": 6,
+        },
+    ),
 ]
 
 
+@asset(io_manager_key="pudl_io_manager")
+def core_eiaaeo__yearly_projected_generation_in_end_use_sectors_by_fuel_type(
+    raw_eiaaeo__electric_power_projections_regional,
+):
+    """Projected generation capacity + gross generation in end-use sectors.
+
+    This includes data that's reported by fuel type and ignores data that's
+    only reported at the system-wide level, such as total generation, sales to
+    grid, and generation for own use.
```

Comment:
why are you removing sales to grid and generation for own use? maybe bc these are ending up in a differnt table bc different pks? i half remember that we talked about this in the schema design but it would be good to add a note here about it.